packages/components/package.json

```diff
@@ -42,11 +42,13 @@
     "@types/testing-library__jest-dom": "^5.14.9",
     "jest": "^27.4.3",
     "react": "18.2.0",
+    "react-dom": "18.2.0",
     "typescript": "^4.9.5",
     "vite": "^4.3.9",
     "vite-plugin-svgr": "^3.2.0"
   },
   "peerDependencies": {
-    "react": "18.2.0"
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
   }
 }
```

Comment:
package version bump?

Reply:
Since I'm making lots of changes in different PRs, I haven't been bumping the version in each PR.

Once Clickable is in here, the changes will be useful to AoU and I'll bump the version.